docs/AboutUs.md

```diff
@@ -53,7 +54,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 <img src="images/johndoe.png" width="200px">
 
 [[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[portfolio](team/tanyyyming.md)]
```

Comment:
Perhaps an accident to change the portfolio but I don't think will affect others